posthog/batch_exports/models.py

```diff
@@ -292,4 +292,10 @@ def workflow_id(self) -> str:
         end_at = self.end_at and self.end_at.isoformat()
         start_at = self.start_at and self.start_at.isoformat()
 
+        if isinstance(end_at, str) and end_at.endswith("+00:00"):
+            end_at = end_at.replace("+00:00", "Z")
```

Comment:
By default, python seems to prefer "+00:00" but Temporal schedules everything with "Z", it's quite annoying.